services/coreai/coreai-mcu/base/patch/kustomize.yaml

```diff
@@ -16,16 +16,19 @@ spec:
   path: ./services/coreai/coreai-mcu/${ENVIRONMENT}/${ENVIRONMENT_ID}
   postBuild:
     substitute:
-      TEAM_NAMESPACE: "coreai-mcu"
-      TEAM_NAME: "coreai-mcu-team"
-      TEAM_RG: "${SERVICES_INFRA_RG}-coreai-mcu"
+      TEAM_NAMESPACE: coreai-mcu
+      TEAM_NAME: coreai-mcu-team
+      TEAM_RG: ${SERVICES_INFRA_RG}-coreai-mcu
       APPCONFIG_MI_CLIENTID: ${APPCONFIG_MI_CLIENTID}
       APPCONFIG_NAME: ${APPCONFIG_NAME}
-      SERVICE_CODE: "coreai-mcu"
-      PLATFORM_DB_ADMIN: "adp-platform-db-aad-admin"
+      SERVICE_CODE: coreai-mcu
+      PLATFORM_DB_ADMIN: adp-platform-db-aad-admin
+      TEAM_CPU_QUOTA: 3000
+      TEAM_MEMORY_QUOTA: 5000
+      TEAM_PODS_QUOTA: __TEAM_PODS_QUOTA__
```

Comment:
@kbabigumira - __TEAM_PODS_QUOTA__ doesn't look correct?

Reply:
Update the value to 50